aiohttp_session/__init__.py

```diff
@@ -175,41 +211,50 @@ def session_middleware(storage):
     return factory
 
 
-def setup(app, storage):
+def setup(app: web.Application, storage: 'AbstractStorage') -> None:
     """Setup the library in aiohttp fashion."""
 
     app.middlewares.append(session_middleware(storage))
 
 
 class AbstractStorage(metaclass=abc.ABCMeta):
 
-    def __init__(self, *, cookie_name="AIOHTTP_SESSION",
-                 domain=None, max_age=None, path='/',
-                 secure=None, httponly=True,
-                 encoder=json.dumps, decoder=json.loads):
+    def __init__(
+        self, *,
+        cookie_name: str = "AIOHTTP_SESSION",
+        domain: Optional[str] = None,
+        max_age: Optional[int] = None,
+        path: str = '/',
+        secure: Optional[bool] = None,
+        httponly: bool = True,
+        encoder: Callable[..., str] = json.dumps,
+        decoder: Callable[..., Dict[Any, Any]] = json.loads
```

Comment:
```suggestion
        decoder: Callable[str, Dict[str, Any]] = json.loads
```

IFAIK the session keys are always strings

Reply:
Fixed :)